appservice/src/main/java/nu/yona/server/rest/TestController.java

```diff
@@ -108,6 +109,24 @@ protected EmailResource instantiateResource(EmailDto email)
 		return createOkResponse(FirebaseMessageDto.createInstance(lastMessage.get()), createFirebaseMessageResourceAssembler());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "lastMessage.isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://10.127.10.107:9000/coding_rules#rule_key=squid%3AS3655)